extensions/interactions/DragAndDropSort/directives/drag_and_drop_sort_interaction_directive.html

```diff
@@ -0,0 +1,63 @@
+<style>
+  .drag-and-drop-sort-form-container {
+    background: #fff;
+    max-width: 100%;
+    padding: 8px;
+  }
+  @media screen and (max-width: 959px) {
+    .drag-and-drop-sort-form-container {
+      width: 100vw;
+    }
+  }
+  .angular-ui-tree-placeholder {
+    width: 200px;
+    height: 70px;
+    border: 2px dashed lightblue;
```

Comment:
Avoid using names for colors (see the [CSS section](https://github.com/oppia/oppia/wiki/Coding-style-guide#css) of our coding style).

Reply:
You may have to break this up so that you can use the rgb(a) or hex values.